spec/icacls_spec.rb

```diff
@@ -0,0 +1,40 @@
+require './icacls_files/icacls_work'
+require 'rspec'
+require './testrail/testrail.rb'
+require './icacls_files/final_result_test'
+require './icacls_files/credentials.rb'
+
+describe Permission do
+  include ResultTesting
+  include Credential
+  before(:context) do
+    @client = TestRail::APIClient.new('https://olhabahno.testrail.io/')
+    @client.user = get_credentials['username']
+    @client.password = get_credentials['password']
+    @users_permission = Permission.new
+  end
+
+  it 'should return RX access' do
+    @users_permission.path = 'C:\\Program Files (x86)\\Entity Framework Tools'
```

Comment:
It's better to use File.join('C:','Program Files', ... )

Reply:
Okay, got it, need to read about it as I didn't work with File.join